lib/match-across-boundaries.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var mutuallyExclusiveDependencyKeys = {
+  dependencies: [
+    'devDependencies'
```

Comment:
Should also include `optionalDependencies`

Reply:
fixed